libraries/libfc/src/crypto/bls_public_key.cpp

```diff
@@ -76,18 +45,18 @@ namespace fc { namespace crypto { namespace blslib {
       return s;

```

Comment:
Should we use a `yield` function in the `to_string()` call in `std::ostream& operator<<(std::ostream& s, const bls_public_key& k)`?

Reply:
I think we should remove `operator<<` for all these types. Going to push up a PR to do that for the non-bls public_key/private_key types now.